ClientTests/PrefsTests.swift

```diff
@@ -91,4 +91,10 @@ class PrefsTests: XCTestCase {
         prefs1.clearAll()
         XCTAssertNil(prefs1.intForKey("foo") as! AnyObject?)
     }
+    
+    func testHomePageSettingForInternalURLs() {
+        let helper = HomePageHelper(prefs: prefs)
+        helper.currentURL = NSURL(string: "http://localhost:6571")
+        XCTAssertNil(prefs.stringForKey(HomePageConstants.HomePageURLPrefKey))
+    }
```

Comment:
Don't think this is the right place to put this -- `PrefsTests` are about testing the prefs API itself. This would probably make more sense in `HomePageTests`.